src/platforms/react-native/troubleshooting.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Troubleshooting
+description: "Troubleshoot and resolve common issues with the React Native SDK."
+sidebar_order: 1000
+---
+
+## Source Maps
+
+When there is an issue with source maps on your event, there will usually be an error dialog at the top of the event page on your dashboard:
+
+> There was 1 problem processing this event: Source code was not found
+
+Common fixes to this are:
+
+- Make sure that both the `release` and `dist` and are set on the event by [setting it in the call to `init`](/platforms/react-native/configuration/releases/#bind-the-version).
+- Make sure that the source maps are correctly uploaded with the exact same `release` and `dist` values as the event.
+- If you set a custom `release` and `dist` make sure that you manually upload the sourcemaps as our automatic sourcemap upload script will not detect the custom values.
```

Comment:
```suggestion
- If you set a custom `release` and `dist`, confirming that you manually uploaded the sourcemaps since our automatic source map upload script will not detect the custom values.
```